pkgs/development/python-modules/bnnumerizer/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "bnnumerizer";
+  version = "0.0.2";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Qd9v0Le1GqTsR3a2ZDzt6+5f0R4zXX1W1KIMCFFeXw0=";
+  };
```

Comment:
Same situation. Create an issue upstream, because the git sources have tests.

Reply:
https://github.com/mnansary/bnUnicodeNormalizer/issues/10